composer.json

```diff
@@ -67,6 +68,7 @@
         "omnipay/2checkout": "~2.0",
         "omnipay/authorizenet": "~2.0",
         "omnipay/buckaroo": "~2.0",
+        "cardgate/omnipay-cardgate": "~2.0",
```

Comment:
Yeah the suggest section seems like a safer idea :) 
Do you want to fix that or should I do it?